server/src/main/java/org/elasticsearch/search/aggregations/AggregatorFactories.java

```diff
@@ -223,13 +223,6 @@ public int countAggregators() {
         return factories.length;
     }
 
-    /**
-     * @return the number of pipeline aggregator factories
-     */
-    public int countPipelineAggregators() {
```

Comment:
Haha was this even used anywhere?  I don't I've seen it before :man_facepalming:

Reply:
Not any more!